NOTICE-3RD-PARTY-CONTENT.md

```diff
@@ -4,68 +4,59 @@
 | Dependency | Version | License |
 |:-----------|:-------:|--------:|
 |aiohttp|3.8.3|Apache 2.0|
-|aiosignal|1.2.0|Apache 2.0|
-|astroid|2.12.11|GNU Lesser General Public License v2 (LGPLv2)|
+|aiosignal|1.3.1|Apache 2.0|
 |async-timeout|4.0.2|Apache 2.0|
 |attrs|22.1.0|MIT|
 |bandit|1.7.4|Apache 2.0|
 |black|22.10.0|MIT|
-|build|0.8.0|MIT|
+|build|0.9.0|MIT|
 |cfgv|3.3.1|MIT|
 |charset-normalizer|2.1.1|MIT|
 |click|8.1.3|New BSD|
-|cloudevents|1.6.1|Apache 2.0|
-|dapr|1.7.0|Apache 2.0|
+|cloudevents|1.7.1|Apache 2.0|
+|dapr|1.8.3|Apache 2.0|
 |deprecation|2.1.0|Apache 2.0|
-|dill|0.3.5.1|New BSD|
 |distlib|0.3.6|Python Software Foundation License|
 |filelock|3.8.0|Public Domain|
 |flake8|5.0.4|MIT|
-|frozenlist|1.3.1|Apache 2.0|
+|frozenlist|1.3.3|Apache 2.0|
 |gitdb|4.0.9|BSD|
 |GitPython|3.1.29|BSD|
 |grpc-stubs|1.24.11|MIT|
-|grpcio|1.49.1|Apache 2.0|
+|grpcio|1.48.2|Apache 2.0|
```

Comment:
why did you downgrade grpcio?

Reply:
It is made by aligning dapr and protobuf by pip-tools